bm25s/tokenization.py

```diff
@@ -35,13 +48,35 @@ def convert_tokenized_to_string_list(tokenized: Tokenized) -> List[List[str]]:
 
 
 def _infer_stopwords(stopwords: Union[str, List[str]]) -> List[str]:
```

Comment:
It would be great to add a notice that the stopwords were taken from NLTK, and link to the exact file/page where they were retrieved!

```suggestion
def _infer_stopwords(stopwords: Union[str, List[str]]) -> List[str]:
    """
    Source of stopwords: ...
    """
```

Reply:
Make sense.